pages/common/git-push.md

```diff
@@ -15,6 +15,10 @@
 
 `git push {{remote_name}} -u {{remote_branch}}`
 
+- Send changes in a given local branch to the named repository's specified remote branch:
```

Comment:
This doesn't make a lot of sense.

Maybe this will read better
```suggestion
- Send changes from a specific local branch to a specified remote branch:
```
Might also want to include that it changes the name of the branch.

Reply:
That suggestion sounds good @CleanMachine1

Reply:
```suggestion
- Send changes from a specific local branch to a specific remote branch:
```